agents/node/vcxagent-cli/storage-agentProvisions/hana/c0daa32b35fe0ec6f987563c3b25b602

```diff
@@ -0,0 +1 @@
+{"key":"agent-provision","value":{"agency_did":"VsKV7grR1BUE29mG2Fm2kX","agency_endpoint":"http://localhost:8080","agency_verkey":"Hezce2UWMZ3wUhVkh2LfKSs8nDzWwzs2Win7EzNN3YaR","genesis_path":"/Users/ABPS823/dev/work/absavcx/agents/node/vcxagent-core/docker.txn","institution_did":"V4SGRU86Z58d6TV7PBUe6f","institution_logo_url":"https://example.org","institution_name":"hana","institution_verkey":"GJ1SzoWzavQYfNL9XkaJdrQejfztN4XqdsiV4ct3LXKL","protocol_type":"4.0","remote_to_sdk_did":"4ZUtYiUneygkCeCoCWugKH","remote_to_sdk_verkey":"2wYGTDvwW9y3s7F1DQYxczgDVSt3FCiBzr6bi1LiQtms","sdk_to_remote_did":"Ue96r3FnNaQqgH7aFE3rvG","sdk_to_remote_verkey":"G4mQWYqDusY774z6MxbQTFqscZz7KmURarrVAeubMggs","wallet_key":"123","wallet_name":"hana"}}
```

Comment:
It seems these two files were commited by accident,  as `storage-*/` doesn't filter them out?